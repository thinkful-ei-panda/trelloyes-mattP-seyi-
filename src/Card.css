.Card {
  border-radius: 4px;
  background-color: hsl(0, 0%, 100%);
  box-shadow: 0 1px 0 hsla(202, 76%, 15%, 0.25);
  padding: 8px;
  color: hsl(202, 54%, 20%);
  margin-bottom: 10px;
}

.Card h3 {
  font-size: 14px;
  margin-bottom: 4px;
}

.Card p {
  margin: 0;
  font-size: 12px;
}

.Card button {
  float: right;
  background-color: hsl(210, 14%, 95%);
  border-radius: 6px;
}