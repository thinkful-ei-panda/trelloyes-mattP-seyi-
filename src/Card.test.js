import React from 'react'
import ReactDOM from 'react-dom'

import renderer from 'react-test-renderer'
import Card from './Card'

describe('<Card />', () => {
	const props = {
		key: 'test_id',
		title: 'Card title',
		content: 'Card content',
	}

	it('should render without crashing', () => {
		const div = document.createElement('div')
		ReactDOM.render(<Card {...props} />, div)
		ReactDOM.unmountComponentAtNode(div)
	})

	it('renders as expected', () => {
		const wrapper = renderer
			.create(<Card {...props} />)
			.toJSON()
		expect(wrapper).toMatchSnapshot()
	})
})
