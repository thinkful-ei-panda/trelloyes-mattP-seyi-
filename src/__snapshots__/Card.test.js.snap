// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Card title
  </h3>
  <p>
    Card content
  </p>
</div>
`;
